docs/build/changelog.rst

```diff
@@ -6,6 +6,16 @@ Changelog
     :version: 0.9.1
     :include_notes_from: unreleased
 
+    .. change::
+        :tags: feature
+
```

Comment:
just an FYI in order for me to avoid merge conflicts, place a file-per-change in docs/build/unreleased